plugins/groovy/src/org/jetbrains/plugins/groovy/lang/completion/GrMethodOverrideCompletionProvider.java

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
OK, no problem =)
Actually if you have no time or don't want to fix all that things I wrote about I can accept the pull request and fix them by myself but it will take a while. So I don't guarantee the feature to be included into the nearest release.